testdata/validate/par1f1.log

```diff
@@ -0,0 +1,36 @@
+Invalid option --
+bde_copy: Extracts data from BDE files
```

Comment:
Maybe, just to reduce size of tests, you could only check for the first line of this error ?
BTW, I believe the par1f1 behavior should be to print the error on stderr, not stdout
(and a convention for unix command is for `--` to have the special meaning of "switches are finished from now on, all the rest are positional arguments").

Reply:
I have updated so that it only checks the first line and because it is an error checking that is printed to stderr not stdout.